qa/integration-tests/src/test/java/io/camunda/zeebe/it/client/command/JobWorkerTest.java

```diff
@@ -82,7 +82,10 @@ void shouldActivateJob(final BiFunction<String, JobWorkerBuilderStep3, JobWorker
                     t.zeebeJobType(jobType)
                         .zeebeJobRetries("5")
                         .zeebeTaskHeader("x", "1")
-                        .zeebeTaskHeader("y", "2"))
+                        .zeebeTaskHeader("y", "2")
+                        .zeebeInput("Hello\nWorld", "newline")
+                        .zeebeInput("Hello\rWorld", "carriageReturn")
+                        .zeebeInput("Hello\tWorld", "tab"))
```

Comment:
:wrench: We could extract the escaped characters in a new test case for better separation.  

:x: We should use input expression instead of static value. The same as in the unit tests.